### PR TITLE
docs: Predictions by Data Element Group [DHIS2-14635]

### PR DESCRIPTION
**Pull request by jimgrace**
Documents Predictions by Data Element Group, see [DHIS2-14635](https://dhis2.atlassian.net/browse/DHIS2-14635).

[DHIS2-14635]: https://dhis2.atlassian.net/browse/DHIS2-14635?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by jimgrace** [comment]
Screenshots of the new documentation from VS Code preview, for ease of review:
Predictions by data element group updated
Predictions by data element group 2

File: src/user/configure-metadata.md [hunk 1]
```diff
@@ -4355,6 +4355,88 @@ sampled. For example:
 
 16. Click **Save**.
 
+### Predictions by Data Element Group { #predictions_by_data_element_group }
+
+You can use a single predictor to operate on all the data elements in a group instead of a different predictor for each data element. This can be used, for example, in logistics management when a data element is used for each commodity and a category option combination is used for each count related to that commodity.
+
+The syntax is:
+
+<pre><code>forEach ?de in :DEG:degUid --> main expression</code></pre>
```
Comment (jbee) [inline src/user/configure-metadata.md]: Is there a particular reason for this formatting or would indent by 4 spaces or 2 tabs result in the same standard "code block"?

File: src/user/configure-metadata.md [hunk 2]
```diff
@@ -4355,6 +4355,88 @@ sampled. For example:
...
+
+The predictor will execute once for each data element in the data element group.
+For each data element, instances of the variable in the main expression are replaced by that data element. The same data element is also used as the predictor output data element.
+The predicted value will be written to that data element using the configured output category option combination.
+
+The predictor must be configured with an output data element, but it is effectively ignored when the predictor is run.
+It is suggested that you configure the predictor with one of the data elements in the data element group that the predictor will use.
+That way you can select a valid output category option combination for that data element.
+
+At the time you create the predictor, the data element group must contain at least one data element of the type that you will use. (The data type of the data element is used during syntax checking of the predictor.)
```
Comment (jbee) [inline src/user/configure-metadata.md]: Maybe word this to not directly address the reader as "you"? E.g. _"At the time the predictor is created, .."._

**Review by jbee — COMMENTED** [review]
I think we usually try not to use wording addressing the reader as "you". I might be wrong on this.

File: src/user/configure-metadata.md (hunk 1 above)
Comment (jimgrace) [reply src/user/configure-metadata.md]: I was just copying from nearby, but 4 spaces works. Thanks, I've changed it.

File: src/user/configure-metadata.md (hunk 2 above)
Comment (jimgrace) [reply src/user/configure-metadata.md]: We address the reader as "you" a lot throughout the document. " you ", with space on either side, appears over 2,000 times in the User Guide. But I agree in this case it's a bit awkward and I've changed it as you suggested. Thanks.

**Review by jbee — APPROVED** [review]

**Review by Philip-Larsen-Donnelly — APPROVED** [review]

**Comment by Philip-Larsen-Donnelly** [comment]
Nice, clear examples, as always. Thanks @jimgrace

**Pull request merged**